common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -296,9 +296,11 @@ def copy_or_rename_transcript(new_name, old_name, item, delete_old=False, user=N
 def get_html5_ids(html5_sources):
     """
     Helper method to parse out an HTML5 source into the ideas
-    NOTE: This assumes that '/' are not in the filename
+    NOTE: This assumes that '/' are not in the filename.
+    Slices each id by 150, restricting too long strings as video names.
     """
-    html5_ids = [x.split('/')[-1].rsplit('.', 1)[0] for x in html5_sources]
+    html5_ids = [x.split('/')[-1].rsplit('.', 1)[0][:150] for x in html5_sources]
```

Comment:
`html5_sources` are basically other html5 URL resources provided for video by course team. Here at this line of code, we are extracting names(html5_ids) for videos from URLs. Why do you think we need to ensure about uniqueness of these html5_ids?